exchange-rates/src/main/java/com/example/exchangeRates/RatesStateFlow.kt

```diff
@@ -0,0 +1,62 @@
+package com.example.exchangeRates
+
+import com.example.exchangeRates.data.RateUi
+import com.ivy.core.domain.action.SharedFlowAction
+import com.ivy.core.domain.action.settings.basecurrency.BaseCurrencyFlow
+import com.ivy.core.persistence.algorithm.calc.Rate
+import com.ivy.core.persistence.algorithm.calc.RatesDao
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.combine
+import kotlinx.coroutines.flow.flatMapLatest
+import kotlinx.coroutines.flow.flowOf
+import javax.inject.Inject
+import javax.inject.Singleton
+
+//returns flow of RatesState which contains manual and automatic exchangeRates list
+@Singleton
+class RatesStateFlow @Inject constructor(
```

Comment:
Check if we have already implemented the same flow in `core:domain`

Reply:
I did check it ,but i did not find a flow which gave manual and automatic exchange rates list separately ,and that's why i implemented it my self ..There was an exchange rates flow though, but it just gave a list of currencies by overriding the automatic ones with manual rates ,and that's not helpful in this case. :(

Reply:
Got it - that's okay 👍 My bad!